x-pack/plugins/cases/public/containers/api.ts

```diff
@@ -153,16 +157,22 @@ export const getSingleCaseMetrics = async (
 
 export const findCaseUserActions = async (
   caseId: string,
+  filterActionType: CaseUserActionTypeWithAll,
+  sortOrder: 'asc' | 'desc',
```

Comment:
I think if we combine the two of them into one object called params it will be easier to extend it in the future.

```
params: {
    type: CaseUserActionTypeWithAll
    sortOrder: UserActionFindRequest['sortOrder']
}
```